snippets/rtt-tracing/rtt-tracing.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
```

Comment:
We already have a `rtt-console` snippet. Could the console redirection be dropped from this snippet?

Reply:
Initially I had trouble getting SystemView tracing to work without redirecting the console, but I took another look after your comment and it appears to work just fine. Pushed an update that drops the redirection from the snippet.